recipe/recipe.yaml

```diff
@@ -9,25 +9,29 @@ package:
 source:
   url: https://github.com/jeremiah-c-leary/vhdl-style-guide/archive/refs/tags/${{ version }}.tar.gz
   sha256: cd394fa6feebe7aa704a41aed9c743baf701d60807f73d191717dcfa188d240c
+  patches:
+    - patches/0001-version-patch.patch
 
 build:
-  number: 1
+  number: 2
   noarch: python
-  script: |
+  script:
     # Create a VERSION file with the correct version
     # because setuptools-git-versioning will not detect
     # the version from the GitHub archive (not a repository,
     # there is no tag).
-    echo "${{ version }}" > VERSION
-    # Patch pyproject to use the dynamic version from the
-    # VERSION file rather than a git tag
-    sed -i 's/template = "{tag}"/version_file = "VERSION"/' pyproject.toml
-    python -m pip install . -vv --no-deps --no-build-isolation
+    - echo ${{ version }} > VERSION
+    - python -m pip install . -vv --no-deps --no-build-isolation
   python:
     entry_points:
       - vsg = vsg.__main__:main
 
 requirements:
+  build:
+    - if: not win
+      then: patch
+    - if: win
+      then: m2-patch
```

Comment:
What issue were you running into that you needed `patch`? I believe `rattler-build` applies the patches before the build step.

Reply:
`conda-build` requires us to specify those in the`build` section, I'm glad if `rattler-build` does not. I will removed them.